### PR TITLE
Change Moderation Dialog from LinearLayout to RelativeLayout

### PR DESCRIPTION
**Pull request by theck13**
### Fix
Allow numerous items to be added to the Hold for Moderation screen without causing the moderation detail text and floating action button to disappear as described in https://github.com/wordpress-mobile/WordPress-Android/issues/4338.

### Test
1. Go to Sites tab.
2. Tap Settings option under Configuration section.
3. Tap More option under Discussion section.
4. Tap Hold for Moderation under Comments section.
5. Tap Add floating action button.
6. Enter unique word.
7. Repeat Step 5 and Step 6 until screen is full.

### CODE REVIEW

File: WordPress/src/main/res/layout/list_editor.xml [hunk 1]
```diff
@@ -20,13 +21,9 @@
         android:textSize="@dimen/text_sz_large"
         android:visibility="gone" />
 
-    <org.wordpress.android.ui.prefs.MultiSelectListView
-        android:id="@android:id/list"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content" />
-
     <org.wordpress.android.widgets.WPTextView
         android:id="@+id/list_editor_footer_text"
```
Comment (khaykov) [inline WordPress/src/main/res/layout/list_editor.xml]: This is header now :)

Comment (theck13) [inline WordPress/src/main/res/layout/list_editor.xml]: Good point.  I'll rename that.

**Comment by khaykov** [comment]
The only other minor thing is a padding situation around the header. Do you think it looks good as is, with large padding at the top, and minimal at the bottom? I tried changing it to have equal padding at top and bottom (using margin_large) but can't tell which one actually looks better :)

**Comment by khaykov** [comment]
Also, just found out that the ability to add more items then the screen can hold (when list becomes scrollable) unveils bunch of issues, specifically with long-press selection after you scroll the list.
I'll file separate issues for them.

**Comment by theck13** [comment]
I didn't notice that at first, but I think `android:paddingEnd="@dimen/margin_extra_large"` was meant to be `android:paddingBottom="@dimen/margin_extra_large"` originally.  I think having equal padding and a divider below the header is less confusing.  I'll add that.

**Comment by theck13** [comment]
> 

That is loosely related to https://github.com/wordpress-mobile/WordPress-Android/issues/4339.  Basically, the list needs to account for offscreen items and implement the `ViewHolder` pattern.

**Pull request merged**